appveyor.yml

```diff
@@ -13,7 +13,10 @@ build: off
 
 install:
   - SET PATH=C:\Ruby%RUBY_FOLDER_VER%\bin;%PATH%
-  - bundle install --retry 5
+  # Bundler downgrade should be removed when the issue is fixed upstream
+#  - gem uninstall bundler -x
+#  - gem install bundler -v "=1.10.6" --no-document
+  - bundle install --retry 5 --jobs=%NUMBER_OF_PROCESSORS%
```

Comment:
We should be able to use `--without` and specify the `vendor` dir via `--path vendor`. Is that invalid?